dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/LinkageCheckerMain.java

```diff
@@ -60,25 +60,73 @@ public static void main(String[] arguments)
         // If JAR files are specified, it's empty.
         ImmutableList<Artifact> artifacts = linkageCheckerArguments.getArtifacts();
 
-        ImmutableSet<LinkageProblem> linkageProblems =
+        Problems problems =
             artifacts.isEmpty()
                 ? checkJarFiles(linkageCheckerArguments)
                 : checkArtifacts(linkageCheckerArguments);
-        if (!linkageProblems.isEmpty()) {
-          System.out.println(
-              "For the details of the linkage errors, see "
-                  + "https://github.com/GoogleCloudPlatform/cloud-opensource-java/wiki/Linkage-Checker-Messages");
-          // Throwing an exception is more test-friendly compared with System.exit(1). The latter
-          // abruptly stops test execution.
-          throw new LinkageCheckResultException(linkageProblems.size());
+
+        Path outputExclusionFile = linkageCheckerArguments.getOutputExclusionFile();
+        // All I/O happens here
```

Comment:
"All output happens here"? The input happens at `checkJarFiles` and `checkArtifacts`.

Reply:
removed